lib/io/rdf/sparql/sparql-query-rdf-source.ts

```diff
@@ -0,0 +1,12 @@
+import {HttpFetch} from "../../fetch/fetch-api";
+import {RdfMemorySource} from "../rdf-source-base";
+import {fetchRdfQuads} from "../http/http-adapter";
+
+export class SparqlQueryRdfSource extends RdfMemorySource {
```

Comment:
Although it may be self explanatory I suggest to add a comment that would explain the difference between SparqlQueryRdfSource and RdfSparqlSource.